packages/swh-plugins.lv2/PKGBUILD

```diff
@@ -0,0 +1,55 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_name=lv2
+pkgname=swh-plugins.lv2
+pkgver=1.0.16
+pkgrel=1
+pkgdesc="LV2 port of Steve Harris' plugins suite"
+arch=(x86_64 aarch64)
+url='http://plugin.org.uk/'
+license=(GPL3)
+groups=(lv2-plugins pro-audio)
+depends=()
+makedepends=(fftw libxslt)
+checkdepends=(lilv lv2lint)
+optdepends=('lv2-host: for running the plugins')
+provides=(swh-plugins)
+source=("$pkgname-$pkgver.tar.gz::https://github.com/swh/$_name/archive/refs/tags/v$pkgver.tar.gz"
+        'fix-segfaults.patch::https://github.com/cbix/swh-lv2/compare/master...fix/activation-malloc.patch')
+sha256sums=('bc24512de6e2fb7a493226e2e01a80ba8462a318b15c3b0fd0cd914b018c3548'
+            '41cf240a090c4bb094ec11b03ce3e2cc520856bac294cc144ab5ad7e30cb68f0')
+
+prepare() {
+  cd $_name-$pkgver
+  patch -p1 -i ../fix-segfaults.patch
+}
+
+build() {
+  cd $_name-$pkgver
+  make
+  # prevent lilv errors during check by removing unbuilt plugins
+  for _dir in plugins/*.lv2; do
+    if [ ! -e $_dir/manifest.ttl ]; then
+      rm -rf $_dir
+    fi
+  done
+}
+
+check() {
+  _lv2path="$srcdir"/$_name-$pkgver/plugins
+  for _plugin in $(LV2_PATH="$_lv2path" lv2ls); do
+    LV2_PATH="$_lv2path":/usr/lib/lv2 lv2lint -Mpack \
```

Comment:
Can you add the `-q` option here?. Makes the check output a lot less cluttered, imo.

Reply:
good idea! 6e35b2b067a3c192b1e5bf151052f8c60ea2feef